### PR TITLE
Converting radar

### PR DESCRIPTION
**Pull request by snallapa**
Radar component with fixed API

### CODE REVIEW

**Comment by jchen006** [comment]
@snallapa very nice job and fantastic work but just some feedback 
Based on this photo 
screen shot 2019-02-13 at 12 27 59 am
(1) Can we so something about the size since it just feels kind of clunky that I need to rescroll 
(2) Is there any possible we can color code the selection against the color that we use for the radar graph I think way way it would be easier for people to follow.

File: app/static/js/components/Experiments/RadarChart/RadarChart.jsx [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+import Radar from "react-d3-radar";
+import TextField from "@material-ui/core/TextField";
+import Select from "react-select";
+import { withStyles } from "@material-ui/core/styles";
+import styles from "./styles";
+import Typography from "@material-ui/core/Typography";
+import Paper from "@material-ui/core/Paper";
+import MenuItem from "@material-ui/core/MenuItem";
+
+function inputComponent({ inputRef, ...props }) {
```
[the remark this answers is not in the document]
Comment (snallapa) [reply app/static/js/components/Experiments/RadarChart/RadarChart.jsx]: yeah.. i was going to do that but forgot haha

**Comment by snallapa** [comment]
new changes made to address issues above. this was much harder than I thought lol but it worked out

**Comment by jchen006** [comment]
Nice job! A little bit slight interaction is off. When I click the "X" should remove the representation on the graph. Second the sizing of the search and select is a little off. Also is there a way to signify when something is 'onHover'? 
screen shot 2019-02-16 at 12 15 09 am

**Comment by snallapa** [comment]
I think spacing can be fixed when that comes up. I fixed the X issue and yeah you can get when you over over points as well with onHover callback for the Radar component

**Pull request merged**